manual/src/refman/extensions/signaturesubstitution.etex

```diff
@@ -100,6 +100,29 @@ module type S = sig
 end [@@expect error];;
 \end{caml_example}
 
+
+Local module type substitutions also allow for the removal of functor
```

Comment:
The use of `removal` sounds potentially confusing since it is not clear what we are removing and when we are removing it. Similarly, extended paths are also allowed in types:
```ocaml
module type s = sig
  type t := Set.Make(String).t
end
```
Maybe we should provide both examples
```
Local substitutions can also be used to give a local name to a type or a module type introduced by a functor application:
```
```ocaml
module type F = sig
  type set := Set.Make(Int).t

  module type Type = sig type t end (* introduced to avoid using `module type of` *)
  module Nest : Type -> sig module type T = Type end

  module type T := Nest(Int).T

  val set: set
  val m : (module T)
end;;
```
?